.tx/config

```diff
@@ -6,7 +6,7 @@ type = STRINGS
 
 [mapbox-navigation-ios.LocalizableStrings]
 file_filter = Sources/MapboxNavigation/Resources/<lang>.lproj/Localizable.strings
-source_file = Sources/MapboxNavigation/Resources/Base.lproj/Localizable.strings
+source_file = Sources/MapboxNavigation/Resources/en.lproj/Localizable.strings
```

Comment:
We probably need to update extract_localizable.sh to put non-IB strings in en.lproj instead of Base.lproj:

https://github.com/mapbox/mapbox-navigation-ios/blob/68f671d7633d8201cd01cc7f20dccc38f43ae4e5/scripts/extract_localizable.sh#L9

Reply:
Fixed